src/st/e_blood_skeleton.h

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "sfx.h"
+
+static u8 anim_walk[] = {
+    0x03, 0x01, 0x06, 0x02, 0x06, 0x03, 0x06, 0x04, 0x06, 0x05,
+    0x06, 0x06, 0x06, 0x07, 0x06, 0x08, 0x03, 0x01, 0x00, 0x00};
+static s16 D_80182624[] = {0, 1, 1, 3, 6, 2, 1, 4, 4, 0};
+static u8 anim_disassemble[] = {
+    0x08, 0x09, 0x08, 0x0A, 0x08, 0x0B, 0x08, 0x0C, 0x08, 0x0D,
+    0x08, 0x0E, 0x08, 0x0F, 0x08, 0x10, 0x08, 0x11, 0x08, 0x12,
+    0x08, 0x13, 0x10, 0x14, 0xFF, 0x00, 0x00, 0x00};
+static u8 anim_reassemble[] = {
+    0x08, 0x14, 0x08, 0x13, 0x08, 0x12, 0x08, 0x11, 0x08, 0x10,
+    0x08, 0x0F, 0x08, 0x0E, 0x08, 0x0D, 0x08, 0x0C, 0x08, 0x0B,
+    0x08, 0x0A, 0x08, 0x09, 0x10, 0x02, 0xFF, 0x00};
+static u8 anim_reassemble_alt[] = {
+    0x03, 0x14, 0x03, 0x13, 0x03, 0x12, 0x03, 0x11, 0x03, 0x10,
+    0x03, 0x0F, 0x03, 0x0E, 0x03, 0x0D, 0x03, 0x0C, 0x03, 0x0B,
+    0x03, 0x0A, 0x03, 0x09, 0x02, 0x02, 0xFF, 0x00};
+
+static s16 timers[] = {64, 40, 96, 56};
```

Comment:
As an aside - this data is present in the NZ0 overlay despite not being used in that stage's version of blood skeleton